receiver/googlecloudspannerreceiver/config.go

```diff
@@ -0,0 +1,114 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package googlecloudspannerreceiver
+
+import (
+	"fmt"
+	"strings"
+
+	"go.opentelemetry.io/collector/receiver/scraperhelper"
+)
+
+type Config struct {
+	scraperhelper.ScraperControllerSettings `mapstructure:",squash"`
+
+	TopMetricsQueryMaxRows int       `mapstructure:"top_metrics_query_max_rows"`
+	Projects               []Project `mapstructure:"projects"`
+}
+
+type Project struct {
+	ID                string     `mapstructure:"project_id"`
+	ServiceAccountKey string     `mapstructure:"service_account_key"`
```

Comment:
Not supporting multiple projects would allow using google default credentials I think which could be more user-friendly

Reply:
I agree but in our use case it is required to have multiple projects.

Reply:
For this and https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/5240/files#r709960450, could you describe the use cases in some more detail?

Reply:
@punya , current configuration options allows you either work with one or multiple projects. Also there was request to us that potential users do not want to have multiple receivers and multiple pipelines configured in scope of one collector instance for Spanner monitoring. They want to have possibility to collect data from all their projects in scope of one receiver instance.